src/components/TodoCard/TodoCard.tsx

```diff
@@ -0,0 +1,64 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hook';
+import { actions as currentTodoActions } from '../../features/currentTodo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoCard: React.FC<Props> = ({ todo }) => {
+  const dispatch = useAppDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  const add = (chtodo: Todo) => {
+    dispatch(currentTodoActions.addTodo(chtodo));
+  };
+
+  function handleEye() {
+    add(todo);
+  }
```

Comment:
The `handleEye` function directly dispatches the `todo` object. Ensure that the `todo` object is immutable or cloned if necessary to avoid unintended side effects.